halyard-deploy/src/main/resources/kubernetes/manifests/exposed-sidecar-service.yml

```diff
@@ -0,0 +1,16 @@
+kind: Service
+apiVersion: v1
+metadata:
+  name: spin-{{ name }}
+  namespace: {{ namespace }}
+  labels:
+    app: spin
+    cluster: spin-{{ cluster }}
+spec:
+  selector:
+    app: spin
+    cluster: spin-{{ cluster }}
+  ports:
+  - protocol: TCP
+    port: {{ port }}
+    targetPort: {{ port }}
```

Comment:
This file is used to have the Redis sentinel service point to the cluster that has the Redis master and Redis sentinel containers.